### PR TITLE
New photometry

### PR DESCRIPTION
**Pull request by mdecleir**
I added the WISE photometry and updated the 2MASS photometry.
The scaling factors were updated accordingly.

WIP: I want to update the IRAC photometry if I can find a catalog.

### CODE REVIEW

**Comment by mdecleir** [comment]
> I added the WISE photometry and updated the 2MASS photometry.
> The scaling factors were updated accordingly.
> 
> WIP: I want to update the IRAC photometry if I can find a catalog.

IRAC photometry has been added. This work is done and can be merged.

File: DAT_files/hd183143.dat [hunk 1]
```diff
@@ -4,11 +4,15 @@ B = 8.080 +/- 0.010
 U = 8.170 +/- 0.010
 R = 5.74  +/- 0.010
 I = 4.79  +/- 0.020
-J = 4.13  +/- 0.020
-K = 3.48  +/- 0.020
+J = 4.179 +/- 0.218 ; JHK ref = IRSA 2MASS All-Sky Point Source Catalog
+K = 3.517 +/- 0.260
```
Comment (karllark) [inline DAT_files/hd183143.dat:8]: Not sure the 2MASS photometry is better.  The 2MASS uncertainties are 10X the existing (literature) photometry.  Ref is probably in Valencic et al. (2004) [I hope].

Comment (karllark) [reply DAT_files/hd183143.dat:8]: Such bright stars saturate in 2MASS, hence the photometry is not nearly as accurate/precise as earlier work that didn't saturate on the stars.

Comment (karllark) [reply DAT_files/hd183143.dat:8]: Found the origin of the photometry (I think).

https://ui.adsabs.harvard.edu/abs/1965ApJ...141..923J/abstract

Comment (mdecleir) [reply DAT_files/hd183143.dat:8]: This paper indeed has the original values (4.13 and 3.48), which are close to the 2MASS values, but no error bars... Should I just leave the 0.02?

Comment (karllark) [reply DAT_files/hd183143.dat:8]: How about keep the original values but increase the uncertainties to 0.1?  Based on your previous comment.

Comment (mdecleir) [reply DAT_files/hd183143.dat]: Ok, done!

**Pull request merged**